openshift/koku-template.yaml

```diff
@@ -371,19 +375,18 @@ parameters:
   name: DATABASE_SERVICE_NAME
   required: true
   value: koku-pgsql
-- description: 'Database engine: postgresql, mysql or sqlite (default).'
+- description: 'Database engine: PostgreSQL'
   displayName: Database Engine
   name: DATABASE_ENGINE
   required: true
   value: postgresql
 - displayName: Database Name
   name: DATABASE_NAME
   required: true
-  value: default
+  value: koku
 - displayName: Database Username
-  from: '[a-zA-Z0-9]{16}'
-  generate: expression
   name: DATABASE_USER
+  value: kokuadmin
```

Comment:
I'd prefer to leave this as generated by default.

You can pass in a set value from the cli (or `init-app.sh`), which overrides the generator.

Reply:
I can definitely do that. What do you see as the gain from a randomized user name? Let's at least append `user-` on the front so it gains some human readability. 

How do I pass in the database user through `init-app.sh`?

Reply:
Okay. The database directly in a pod in OpenShift is likely a stopgap while developing, switching to something hosted (like RDS if we were to stay with RDBMS/Postgres). I have the local db make command to give me consistent db access, which I view as essential for development. I am interested to hear your thoughts on tooling. 

Currently the `-p` flag is reserved for modifying the port in init-app.sh.

Reply:
I think you're misunderstanding which `-p` I'm referring to.

Inside `init-app.sh` is the `oc new-app` command. Add a `--param KEY=VALUE` to the `oc new-app`. 

If you want to add a flag to `init-app.sh` too, just pick a different letter or use a positional argument, which doesn't require a flag. It should "just work". e.g. `init-app.sh -x foo positional=bar`